app/Controllers/Api/V1/CategoriesController.php

```
@@ -28,52 +35,117 @@ public function __construct()
-        return $this->fail('Not Implemented', 501);
+        $payload = $this->request->getJSON(true);
+
+        if (!$payload) {
+            return $this->failValidationErrors('Invalid payload');
+        }
+
+        if (!$this->validateData($payload, $this->rules)) {
+            return $this->failValidationErrors($this->validator->getErrors());
+        }
+
+        $payload['tenant_id'] = auth()->user()->tenant_id;
```

PHPStan (2.1.46)

[error] Access to an undefined property CodeIgniter\Shield\Entities\User::$tenant_id.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)